README.md

```diff
@@ -1,59 +1,48 @@
-<!-- Copyright (c) 2018 Nikunj Gupta -->
-
-
 # Rescue Net
 
-## Abstract
-
-Predicting natural disasters has always been a challenge for researchers around 
-the world. The number of such occurrences are also increasing by the day. Until 
-we find a way to make such deductions deterministic, we will require a robust 
-way to help victims of these disasters.
-
-In this project, we propose a robust way to make communication with victims 
-suffering in the disaster. We aid them by through communication with the 
-concerned authorities responsible for the rescue operation. We present a 
-drone-based solution for communicating with mobile devices.
+Current instruments and methods fail at predicting natural calamities like
+earthquake. Tsunami predictions are made about a few hours before which leads
+to problems in evacuation of the targetted area. Due to such shortcomings,
+authorities are not able to take adequate measures to prevent the loss of life.
+A lot of research is undergoing to try and predict such disasters in a timely
+manner. Until then we require a system to minimize the loss of lfe and making
```

Comment:
correct the spelling of life

Reply:
It has been corrected. Thanks for pointing it out!